Description: Fix build with newer Sphinx
 This is a partial cherry-pick of 83407ffc7acc00cc025b9f6ed063add9ab9f9bcc.
Origin: https://gitlab.haskell.org/ghc/ghc/-/commit/83407ffc7acc00cc025b9f6ed063add9ab9f9bcc
Bug-Debian: https://bugs.debian.org/995429

Index: b/docs/users_guide/conf.py
===================================================================
--- a/docs/users_guide/conf.py
+++ b/docs/users_guide/conf.py
@@ -77,13 +77,13 @@ htmlhelp_basename = 'GHCUsersGuide'
 latex_elements = {
     'inputenc': '',
     'utf8extra': '',
-    'preamble': '''
+    'preamble': r'''
 \usepackage{fontspec}
 \usepackage{makeidx}
 \setsansfont{DejaVu Sans}
 \setromanfont{DejaVu Serif}
 \setmonofont{DejaVu Sans Mono}
-\setlength{\\tymin}{45pt}
+\setlength{\tymin}{45pt}
 ''',
 }
 
